docs/source/markdown/podman-remote.1.md

```diff
@@ -31,29 +33,19 @@ Remote connection name
 
 Print usage statement
 
-**--log-level**=*level*
-
-Log messages above specified level: debug, info, warn, error (default), fatal or panic
-
-**--port**=*integer*
-
-Use an alternative port for the ssh connections.  The default port is 22
+**--identity**=*path*
 
-**--remote-config-path**=*path*
+Path to ssh identity file. If the identity file has been encrypted, podman prompts the user for the passphrase.
+If no identity file is provided and no user is given, podman defaults to the user running the podman command.
+Podman prompts for the login password on the remote server.
 
-Alternate path for configuration file
-
-**--remote-host**=*ip*
-
-Remote host IP
-
-**--syslog**
+**--log-level**=*level*
```

Comment:
I thought this was a global option and only documented in podman(1)?

Reply:
podman(1) isn't included in the remote package. podman-remote(1) will be renamed to podman(1) while generating docs for remote.